programs/zstdcli.c

```diff
@@ -136,6 +136,8 @@ static int usage_advanced(const char* programName)
     DISPLAY( " -q     : suppress warnings; specify twice to suppress errors too\n");
     DISPLAY( " -c     : force write to standard output, even if it is the console\n");
     DISPLAY( " -l     : print information about zstd compressed files \n");
+    DISPLAY( " --output-dir-flat dir    : result(s) stored into toplevel of target existing directory `dir`. If multiple files have the same name, -f will cause the final file to be the output, otherwise the first will be output. Requires POSIX >= 2001.12 \n");
```

Comment:
Rather than describe the platform requirements here, simply choose whether to display this line at all depending on whether zstd was build with support for this functionality.